File: PHP/TestCases/AndroidAppTest.php
<?php
use PHPUnit\Framework\TestCase;
include('Utils/Driver.php');
include('Utils/DesiredCapabilities.php');
include('Objects/AndroidApp.php');

class AndroidAppTest extends TestCase {
  public $driver_, $cap, $testApp;

  public function setUp() {
    $this->cap = GetAndroidAppDesiredCapability();
    $this->driver_ = CreateDriver($this->cap);
    $this->testApp = new ApiDemos($this->driver_);
  }

  public function test_ClickOnApp() {
    print "\n***** Should be able to click on 'App' category *****";
    $this->assertTrue($this->testApp->clickOnAppCategory());
  }

  public function tearDown() {
    QuitDriver($this->driver_);
  }
}
?>

File: Ruby/Testcases/AndroidAppTest.rb
require 'selenium-webdriver'
require 'test/unit'
require_relative '../Objects/AndroidApp'
require_relative '../Utils/Driver'
require_relative '../Utils/DesiredCapabilities'

class AndroidAppTest < Test::Unit::TestCase

  def setup
    @cap = getAndroidAppDesiredCapability
    @driver = CreateDriver @cap
    @testApp = AndroidApp.new @driver
    @expectedCategoryName = 'Activity'
  end

  def test_should_show_activity_category
    assert_equal @expectedCategoryName, @testApp.findActivityCategory
  end

  def teardown
    QuitDriver @driver
  end

end
